src/interfaces/log/ruuvi_interface_log.c

```diff
@@ -346,16 +345,6 @@ void ri_log (const ri_log_severity_t severity,
     return;
 }
 
-/**
```

Comment:
Why this was removed?

Reply:
Incorrect issue previously fixed. Returned in latest commit.